src_py/__init__.py

```diff
@@ -184,6 +226,7 @@ def warn_unwanted_files():
 
     files = py_files + extension_files
 
+    # Creates a list of unwanted files
```

Comment:
A comment which just describes what a simple piece of code does is not very helpful for the reader.
Here is a longer explanation of why: https://realpython.com/python-comments-guide/#python-commenting-worst-practices